Recolor.php

```diff
@@ -56,4 +56,14 @@ public function image($imageName, $newFilename)
         imagepng($image, $this->imageFolder . $newFilename);
         imagedestroy($image);
     }
-}
\ No newline at end of file
+}
+
+/**
+ * Let's actually run the code!
+ */
+
+// Instanciate a new instance of the class
+$recolor = new Recolor();
+
+// Call the class method. As it returns void, we don't need to assign it's return value.
+$recolor->image('etsy.png', 'atomic.png');
```

Comment:
Here we just call the class method. As it's visibility keyword is `public` we're allowed to call it from outside the class. So we've just run that class method, with some inputs!